application-services/custom/camera-management/go.mod

```diff
@@ -8,70 +8,70 @@ module github.com/edgexfoundry/edgex-examples/application-services/custom/camera
 go 1.18
 
 require (
-	github.com/IOTechSystems/onvif v0.0.2-0.20220301065030-7cf2dd734897
-	github.com/edgexfoundry/app-functions-sdk-go/v2 v2.3.0
-	github.com/edgexfoundry/go-mod-bootstrap/v2 v2.3.0
-	github.com/edgexfoundry/go-mod-core-contracts/v2 v2.3.0
+	github.com/IOTechSystems/onvif v0.1.6
+	github.com/edgexfoundry/app-functions-sdk-go/v3 v3.0.0-dev.66
```

Comment:
@presatish , the SDK version can now be bumped to v3.0.0. Make tidy will bump the others